api/lib/domain/models/CertificationAnswerStatusChangeAttempt.js

```diff
@@ -0,0 +1,36 @@
+module.exports = class CertificationAnswerStatusChangeAttempt {
+  constructor(questionNumber, status) {
+    this.questionNumber = questionNumber;
+    this.status = status;
+  }
+
+  static changed(questionNumber) {
+    return new CertificationAnswerStatusChangeAttempt(questionNumber, CertificationAnswerStatusChangeStatus.CHANGED);
+  }
+
+  static failure(questionNumber) {
```

Comment:
Pour mettre tout en cohérence (participe passé - change, skipped..)
```suggestion
  static failed(questionNumber) {
```

Reply:
Done